examples/EksempelSn2007.ttl

```diff
@@ -50,7 +50,7 @@
 <fastlandsNorge2009> a skos:Concept ; . # for å illustrere bruk av xkos:variant
 
 ### -------------------------------------------
-### klassifikasjonsnivåer (xkos:ClassificationLevel)
+### klassifikasjonsnivåer (xkos:ClassificationLevel som er siubklasse av skos:Collection)
 ### -------------------------------------------
```

Comment:
i for mye i subklasse

Reply:
Ja, dessverre. Tar det i neste oppdatering (som jeg gjør nokså "hyppig" i denne oppbyggingsfasen).